src/components/EditableText/EditableText.test.tsx

```diff
@@ -240,39 +210,62 @@ describe('<EditableText />', () => {
             expect(component).toMatchSnapshot();
           });
         });
+      });
+    });
+
+    describe('when focusableRefs are provided', () => {
+      let clickableFocusable: Focusable;
+      let textAreaFocusable: Focusable;
+
+      beforeEach(() => {
+        // Populate the mock focus listeners. We need to force the control into
+        // edit mode, and then close it back up again, just so that we can inject
+        // both of these ahead of testing.
+
+        clickableFocusable = { focus: jest.fn() };
+        const setClickableFocusable = component.find(Clickable).prop('focusableRef') as SetFocusableRef;
+        setClickableFocusable(clickableFocusable);
+
+        // Force the component open...
+        component.find(Clickable).simulate('click', fakeEvent);
+
+        // Now we can hook up the focusable ref for the TextArea...
+        textAreaFocusable = { focus: jest.fn() };
+        const setTextAreaFocusable = component.find(TextField).prop('focusableRef') as SetFocusableRef;
```

Comment:
Comment and variable names say `TextArea` but component is actually `TextField`. Is this intentional?